### ISSUE TITLE
Newly Created Chapter is highlighted without becoming current so Study, Games and Exercises show not what user expects

### ISSUE THREAD

**Issue by eradecka** [czbar/ChessForge#293]
I have 2 Chapters and each has 1 Exercise (Test Exercise in Chapter 1 and Test Exercise 2 in Chapter 2) 

![image](https://user-images.githubusercontent.com/110824781/198555949-dbc36339-05db-4052-9c2a-dec2beaf36dd.png)

While Chapter 2 is highlighted i run Create New Chapter:

![image](https://user-images.githubusercontent.com/110824781/198556339-0d89f268-6dbe-4e5e-a1e2-66018b5d60ba.png)

New Chapter 2 is created as highlighted as current:

![image](https://user-images.githubusercontent.com/110824781/198556629-f1821866-1bf7-40c0-a92a-afb2af9a1d9d.png)

When I go Exercises tab now I can see Exercise from Chapter 2 (which was previously current)

![image](https://user-images.githubusercontent.com/110824781/198556814-81d8af80-7cbe-4834-a38c-351ffe80e2b2.png)

When I go to Study Tab it shows me Chapter 2:

![image](https://user-images.githubusercontent.com/110824781/198556898-1263e64c-a9b1-4f93-a3cb-677fa83a4839.png)

So basically new chapter is highlighted but does not become current

**Comment by eradecka** [comment]
Games and Exercises tabs are fixed now and show nothing however Study tab still shown data from previously current chapter. See below, Chapter 2 is highlighted that I just created:

![image](https://user-images.githubusercontent.com/110824781/198755419-413aff05-32bb-427f-b860-4d0c4f7ff145.png)

But study tree shows Chapter 1:

![image](https://user-images.githubusercontent.com/110824781/198755446-23ce53ba-0d9d-4c17-a4ac-3d13116d1a00.png)
